### PR TITLE
Add publish data for features/dataflow branch

### PR DESCRIPTION
**Pull request by mavasani**
<details><summary>Ask Mode template not completed</summary>

### Customer scenario

What does the customer do to get into this situation, and why do we think this
is common enough to address for this release.  (Granted, sometimes this will be
obvious "Open project, VS crashes" but in general, I need to understand how
common a scenario is)

### Bugs this fixes

(either VSO or GitHub links)

### Workarounds, if any

Also, why we think they are insufficient for RC vs. RC2, RC3, or RTW

### Risk

This is generally a measure our how central the affected code is to adjacent
scenarios and thus how likely your fix is to destabilize a broader area of code

### Performance impact

(with a brief justification for that assessment (e.g. "Low perf impact because no extra allocations/no complexity changes" vs. "Low")

### Root cause analysis

How did we miss it?  What tests are we adding to guard against it in the future?

### How was the bug found?

(E.g. customer reported it vs. ad hoc testing)

### Test documentation updated?

If this is a new non-compiler feature or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.

</details>

### CODE REVIEW

File: build/config/PublishData.json [hunk 1]
```diff
@@ -49,6 +49,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn-nonnull/vsix/upload" ],
             "channels": [ "nonnull" ]
         }
+        "features/dataflow": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.8.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
```
Comment (jmarolf) [inline build/config/PublishData.json]: don't we want a separate feed for this?

**Comment by jmarolf** [comment]
This PR needs to be against `features/dataflow` instead of `master` if publish is going to work.

File: build/config/PublishData.json (hunk 1 above)
Comment (jmarolf) [reply build/config/PublishData.json]: this is resolved by https://github.com/dotnet/roslyn/pull/25922

File: build/config/PublishData.json [hunk 2]
```diff
@@ -49,6 +49,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn-nonnull/vsix/upload" ],
             "channels": [ "nonnull" ]
         }
```
Comment (jaredpar) [inline build/config/PublishData.json]: You need a comma following this close `}`

Comment (mavasani) [reply build/config/PublishData.json]: Thanks, addressed.

File: build/config/PublishData.json [hunk 3]
```diff
@@ -48,6 +48,13 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn-nonnull/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn-nonnull/vsix/upload" ],
             "channels": [ "nonnull" ]
+        },
+        "features/dataflow": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.8.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```
Comment (jasonmalinowski) [inline build/config/PublishData.json]: Historically we've provided a separate feed, as it's not uncommon for partners to grab packages from our main feed without realizing it. Can we create a new feed? You should just be able to go to dotnet.myget.org and make one yourself.

Comment (jaredpar) [reply build/config/PublishData.json]: These packages have a different pre-release tag hence same feed should be fine.

Comment (mavasani) [reply build/config/PublishData.json]: @jasonmalinowski Any further concerns here?

**Comment by mavasani** [comment]
@dotnet-bot retest windows_debug_vs-integration_prtest please

**Comment by mavasani** [comment]
@dotnet-bot retest windows_release_unit32_prtest please

**Pull request merged**